src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/AlleleSubsettingUtils.java

```diff
@@ -309,10 +309,6 @@ public static List<Allele> calculateMostLikelyAlleles(final VariantContext vc, f
         }
 
         final double[] likelihoodSums = calculateLikelihoodSums(vc, defaultPloidy, allHomRefData);
```

Comment:
Can you confirm that `calculateLikelihoodSums()` will be ok even though `allHomRefData` will be false when there are unnormalized PLs? In particular, I am concerned about this line in `calculateLikelihoodSums()`:

```
            final int indexOfMostLikelyVariantGenotype = MathUtils.maxElementIndex(glsVector, allHomRefData ? 1 : 0, glsVector.length);
```